docs/Contributing.asciidoc

```diff
@@ -431,10 +489,245 @@ Also find more details in
   look at the
   https://github.com/Martchus/openQA-helper[openQA-helper repository].
 
+[[quick-container-setup]]
+=== Quick container development setup
+
+==== Requirements
+
+In these examples we use the following tools which are all in openSUSE Leap:
+
+* https://podman.io/[`podman`]
+* https://en.opensuse.org/Distrobox[`distrobox`]
+
+You can use similar tools like `docker`, `toolbox` etc. to achieve the same.
+
+==== Intro
+
+These instructions are ready to use without any changes. You only might want to
+change the `OPENQA_BASEDIR` location.
+
+You will run two containers:
+
+* A separate postgres container (so you don't have to deal with the database
+  when updating your dev container)
+* The openQA environment in an openSUSE Leap or Tumbleweed container, which
+  both can use the same postgres container.
+  In this example we will be setting up and using the Leap container.
+
+`distrobox`, for example, can be used, to take care of network, home directory,
+permissions, and graphical environment.
+
+Some common settings we are using here:
+
+* `OPENQA_BASEDIR=$HOME/openqadev.leap`
+* Database name: `openqa-local`
+* Database user: `geekotest`
+* Database container name: `postgres-openqa`
+
+==== Environment Variables
+
+Create a file that you can `source` whenever you want to work in this
+environment:
+
+File: `~/localopenqa.leap.sh`:
+[source,sh]
+----
+#!/bin/bash
+
+# The directory where openQA operates in and creates everything in:
+# * git repos
+# * openQA assets, pool directory, testresults
+# * configuration
+
+export OPENQA_BASEDIR=$HOME/openqadev.leap
+
+# The rest of the variables can likely stay like this
+
+# If you want to import database dumps from elsewhere
+#export SQL_DUMP_DIR=$HOME/sqldumps
+
+export OPENQA_CONFIG=$OPENQA_BASEDIR/config
+export OPENQA_REPOS="$OPENQA_BASEDIR/git"
+PATH=$OPENQA_REPOS/openQA/scripts:$PATH
+
+# ----------- CPAN ------------
+# Whenever necessary, you can install additional modules from CPAN like:
+# cpanm -l $OPENQA_BASEDIR/perl5 Some::Module@1.23
+export PERL5LIB=$OPENQA_BASEDIR/perl5/lib/perl5
+PATH=$OPENQA_BASEDIR/perl5/bin:$PATH
+
+export OPENQA_KEY=1234567890ABCDEF
+export OPENQA_SECRET=1234567890ABCDEF
+----
+
+==== Postgres Container
+
+To be able to keep the database when stopping the container, we create a
```

Comment:
niptick: would it be better if you say `To be able to reuse the database..`?

Reply:
I rephrased it